roles/openshift_master/tasks/main.yml

```diff
@@ -24,10 +24,10 @@
   firewalld: port=4001/tcp permanent=true state=enabled
 
 - name: Open firewalld port for OpenShift
```

Comment:
@akram Nice, didn't think about using nested there...  I'm looking forward to the immediate flag being introduced with ansible 1.9 to remove the need to issue 2 firewalld statements for each rule